packages/core/mesh/network-manager/src/network-manager.ts

```diff
@@ -106,8 +108,8 @@ export class NetworkManager {
     return this._signalManager;
   }
 
-  get networkMode() {
-    return this._networkMode;
+  get connectionState() {
```

Comment:
Just note, we may extend this in the future to return an object with other props.